### DISCUSSION TITLE
Subdomain compatibilty?

### DISCUSSION THREAD

**Discussion by arladmin** [plausible/analytics#671] [Anything Else]
Hi, i have a bunch of web-apps that i would like to track.
Their urls are in the following format:
https://abc.example.com/app1
https://abc.example.com/app2
https://abc.example.com/app3

I have tried adding the websites in Plausible with: "abc.example.com/appx" format, but that doesn't work.

Even though i can see in my web-app that the Plausible (hash) script has been loaded as well as the events are being fired.

Is this is a bug? Or, do i need to do something else to accommodate my sub-domains in some other way?

Thanks

**Comment by metmarkosaric** [comment]
Thanks @arladmin! Both folders and subdomains should work out of the box.

**Comment by ukutaht** [comment]
> I have tried adding the websites in Plausible with: "abc.example.com/appx" format, but that doesn't work.

Can you specify what problems come up when you do this? We have many people using github pages and the system is designed to work with separating subfolders in to their own dashboards.
